pkg/connector/slack/v0/config/definition.json

```diff
@@ -0,0 +1,46 @@
+{
+  "available_tasks": [
+    "TASK_READ_MESSAGE",
+    "TASK_WRITE_MESSAGE"
+  ],
+  "custom": false,
+  "documentation_url": "",
+  "icon": "assets/xxx.svg",
+  "icon_url": "",
+  "id": "slack",
+  "public": true,
+  "spec": {
+    "connection_specification": {
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "additionalProperties": false,
+      "properties": {
+        "api_key": {
```

Comment:
Please try to align the naming with the vendor. Slack is using `token`, so let's use `token` here.